app/views/content/non-uk-teachers/return-to-england-after-teaching-overseas/_check-your-qualifications.md

```diff
@@ -1,8 +1,8 @@
-You will need [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) to teach in a maintained school in England.
+[Qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) is a legal requirement to teach in many English schools, and considered desirable for teachers in the majority of schools in England.
 
-Even where QTS is not a legal requirement, for example in academies and free
-schools, many schools use it to assess the quality of candidates for teaching
-jobs.
+Schools in England where teachers can be employed without QTS include [academies](https://www.gov.uk/types-of-school/academies), [free schools](https://www.gov.uk/types-of-school/free-schools) and [private schools](https://www.gov.uk/types-of-school/private-schools).
```

Comment:
```suggestion

```
Shouldn't really be having one word links for accessibility - can we add a line linking to the types of schools instead? (See added comment down below.)